Gemfile

```diff
@@ -5,10 +5,10 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.21'
+gem 'rails', '4.0.12'
```

Comment:
This is quite concerning, maybe it ought to be discussed on -dev?  Obviously, the available SCLs will be a factor too...